pluribus/games/short_deck/state.py

```diff
@@ -159,9 +165,9 @@ def apply_action(self, action_str: Optional[str]) -> ShortDeckPokerState:
                 f"type {type(action)}."
             )
         # Update the new state.
-        n_skips = new_state._skip_counter
-        new_state._history = new_state._history + ["skip"] * n_skips
-        new_state._history.append(str(action))
+        skip_actions = ["skip" for _ in range(new_state._skip_counter)]
+        new_state._history[new_state.betting_stage] += skip_actions
```

Comment:
I need to check this, but this is potentially awkward now. If the action is on the flop, for example, and player 2 is out of the hand, we would expect a ['call', 'call', 'skip'] sequence to be stored under flop. But if 'skip' is stored in the skip counter, I think the ```betting_stage``` would update and then the 'skip' would append to the turn. 

I will check. But, I also don't see right now how we set the ```_first_move_of_current_round``` when we increment the round yet, so we could just reference a ```_previous_betting_stage``` if ```_first_move_of_current_round``` is equal to True.

Reply:
This was a non-issue. It turns out that we don't append skips if the round action is finished. So, if player 2 has folded previously, and then on the flop we see ['call', 'call'] for players 0 and 1, the skips are not added to the end of the action sequence. 

This is just semantics, but flagging it in case you care. My test now should work in either case I believe.